erpnext/manufacturing/report/cost_of_poor_quality_report/cost_of_poor_quality_report.py

```diff
@@ -20,30 +19,39 @@ def get_data(report_filters):
 	if operations:
 		operations = [d.name for d in operations]
 		fields = ["production_item as item_code", "item_name", "work_order", "operation",
-			"workstation", "total_time_in_mins", "name", "hour_rate", "serial_no", "batch_no"]
+			"workstation", "total_time_in_mins", "name", "hour_rate", "serial_no",
+			 "batch_no","(hour_rate) * ((total_time_in_mins) / 60.0) as operating_cost"]
 
 		filters = get_filters(report_filters, operations)
+		time_filters = get_timelog_filters(report_filters)
 
-		job_cards = frappe.get_all("Job Card", fields = fields,
-			filters = filters)
-
-		for row in job_cards:
-			row.operating_cost = flt(row.hour_rate) * (flt(row.total_time_in_mins) / 60.0)
-			data.append(row)
+		data = frappe.db.sql("""SELECT {fields} FROM `tabJob Card` tjc {filters}
```

Comment:
Translate this to use query builder now 😅 

We are gearing up for Postgres support, so new raw queries won't be allowed from now on. 

https://frappeframework.com/docs/user/en/api/query-builder

Reply:
Okay. I will translate the query to use QB.